config/config.go

```diff
@@ -165,6 +168,9 @@ func ReloadConfig() error {
 	if c.RepositoryScanInterval < 0 {
 		c.RepositoryScanInterval = 0
 	}
+	if c.MetricsTopFilesRetention < 1 {
+		return fmt.Errorf("Invalid retention duration in days")
```

Comment:
What if you don't care about this feature? I would allow 0 and disable the feature entirely.